docs/changelogs/CHANGELOG-1.3.md

```diff
@@ -54,8 +54,11 @@
 
 ### Deprecated
 
+- Vault and Istio components are deprecated and will be removed in the next version
+
 ### Breaking changes
 
 - PgBouncer available only as Kubernetes service
+- Upgrading Kubernetes to the latest available version 1.22 breaks Vault and Istio components which are deprecated
```

Comment:
Put to known issues that vault and istio are not known not to work anymore, and change the language from deprecated as being considered for deprecation. As we don't have sight on who is using this we cant straight up deprecate it for now.